base/stats.go

```diff
@@ -1322,167 +1418,167 @@ func (d *DbStats) initDatabaseStats() error {
+	resUtil.SyncFunctionTime, err = NewIntStat(SubsystemDatabaseKey, "sync_function_time", StatUnitNanoseconds, SyncFunctionTimeDesc, StatVersionAdded3dot1dot0, StatStabilityCommitted, labelKeys, labelVals, prometheus.CounterValue, 0)
 	if err != nil {
 		return err
 	}
-	resUtil.SyncFunctionExceptionCount, err = NewIntStat(SubsystemDatabaseKey, "sync_function_exception_count", SyncFunctionExceptionCountDesc, labelKeys, labelVals, prometheus.CounterValue, 0)
+	resUtil.SyncFunctionExceptionCount, err = NewIntStat(SubsystemDatabaseKey, "sync_function_exception_count", StatUnitNoUnits, SyncFunctionExceptionCountDesc, StatVersionAdded3dot1dot0, StatStabilityCommitted, labelKeys, labelVals, prometheus.CounterValue, 0)
 	if err != nil {
 		return err
 	}
-	resUtil.NumReplicationsRejectedLimit, err = NewIntStat(SubsystemDatabaseKey, "num_replications_rejected_limit", NumReplicationsRejectedLimitDesc, labelKeys, labelVals, prometheus.CounterValue, 0)
+	resUtil.NumReplicationsRejectedLimit, err = NewIntStat(SubsystemDatabaseKey, "num_replications_rejected_limit", StatUnitNoUnits, NumReplicationsRejectedLimitDesc, StatVersionAdded3dot1dot0, StatStabilityNoStability, labelKeys, labelVals, prometheus.CounterValue, 0)
 	if err != nil {
 		return err
 	}
-	resUtil.NumPublicRestRequests, err = NewIntStat(SubsystemDatabaseKey, "num_public_rest_requests", NumPublicRestRequestsDesc, labelKeys, labelVals, prometheus.CounterValue, 0)
+	resUtil.NumPublicRestRequests, err = NewIntStat(SubsystemDatabaseKey, "num_public_rest_requests", StatUnitNoUnits, NumPublicRestRequestsDesc, StatVersionAdded3dot1dot0, StatStabilityNoStability, labelKeys, labelVals, prometheus.CounterValue, 0)
 	if err != nil {
 		return err
 	}
-	resUtil.TotalSyncTime, err = NewIntStat(SubsystemDatabaseKey, "total_sync_time", TotalSyncTimeDesc, labelKeys, labelVals, prometheus.CounterValue, 0)
+	resUtil.TotalSyncTime, err = NewIntStat(SubsystemDatabaseKey, "total_sync_time", StatUnitSeconds, TotalSyncTimeDesc, StatVersionAdded3dot1dot0, StatStabilityNoStability, labelKeys, labelVals, prometheus.CounterValue, 0)
 	if err != nil {
 		return err
 	}
-	resUtil.ImportProcessCompute, err = NewIntStat(SubsystemDatabaseKey, "import_process_compute", ImportProcessComputeDesc, labelKeys, labelVals, prometheus.CounterValue, 0)
+	resUtil.ImportProcessCompute, err = NewIntStat(SubsystemDatabaseKey, "import_process_compute", StatUnitNoUnits, ImportProcessComputeDesc, StatVersionAdded3dot1dot0, StatStabilityNoStability, labelKeys, labelVals, prometheus.CounterValue, 0)
 	if err != nil {
 		return err
 	}
-	resUtil.SyncProcessCompute, err = NewIntStat(SubsystemDatabaseKey, "sync_process_compute", SyncProcessComputeDesc, labelKeys, labelVals, prometheus.CounterValue, 0)
+	resUtil.SyncProcessCompute, err = NewIntStat(SubsystemDatabaseKey, "sync_process_compute", StatUnitNoUnits, SyncProcessComputeDesc, StatVersionAdded3dot1dot0, StatStabilityNoStability, labelKeys, labelVals, prometheus.CounterValue, 0)
```

Comment:
Do we want to differentiate `StatUnitNoUnits` and `TODO`/`Unspecified`?

This stat _technically does_ have a unit (albeit a weird one and not directly relevant to customers: millisecond bytes)

Reply:
Per spec we want to follow the units given here - https://prometheus.io/docs/practices/naming/#base-units (or very close, eg that says seconds but we can use nanoseconds) but we don't want to force a bunch of units in where it doesn't really fit. Talked with Malarky about this